### DISCUSSION TITLE
Possible to load without caching?

### DISCUSSION THREAD

**Discussion by 2000Nic** [zauberzeug/nicegui#588] [Q&A] [answered]
When using different images with the same name our browsers cache the image meaning the user has to update the page without cache to see their actual image. Is there a way to prevent this? Could there be a parameter to change in `ui.run()`?

**Comment by falkoschindler** [comment, marked as answer]
Sure, you can do that. One common approach is to add some dummy argument to the image source:

```py
image = ui.image('https://picsum.photos/200/300.jpg').classes('w-40')
ui.button('Update').on('click', lambda: image.set_source(f'https://picsum.photos/200/300.jpg?t={time.time()}'))
```

See https://stackoverflow.com/a/729623/3419103 for a more detailed explanation.

**Reply by rodja** [reply]
We also do this in the opencv webcam demo:

https://github.com/zauberzeug/nicegui/blob/997a404ef59fa9bad2c812fc77824f922cc87429/examples/opencv_webcam/main.py#L46-L49

**Comment by eddie3ruff** [comment]
is there a way to do this timestamp trick to base64 images? My browser is still caching those to memory for my interactive images.
